.github/workflows/ci.yml

```diff
@@ -12,27 +12,37 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
+      - name: Checkout source
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
```

Comment:
Weird...?

Reply:
For some reason the submodules checkout didn't work with the other version...? This is the version of the action we're using in `tbdex-js`